src/index.js

```diff
@@ -1,19 +1,19 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import MultiGrid from 'react-virtualized/dist/commonjs/MultiGrid';
-import classNames from 'classnames';
-import { withStyles } from '@material-ui/core/styles';
-import Table from '@material-ui/core/Table';
-import TableBody from '@material-ui/core/TableBody';
-import TableCell from '@material-ui/core/TableCell';
-import TableFooter from '@material-ui/core/TableFooter';
-import TablePagination from '@material-ui/core/TablePagination';
-import TableSortLabel from '@material-ui/core/TableSortLabel';
-import Draggable from 'react-draggable';
-
-import { calcColumnWidth } from './utils';
-
-const FOOTER_BORDER_HEIGHT = 1;
+import React from 'react'
+import PropTypes from 'prop-types'
+import MultiGrid from 'react-virtualized/dist/commonjs/MultiGrid'
+import classNames from 'classnames'
+import {
+  Table,
+  TableCell,
+  TableFooter,
+  TablePagination,
+  TableSortLabel,
+  withStyles
+} from '@material-ui/core'
```

Comment:
I typically import these individually to reduce bundle size, but webpack are typically pretty good nowadays.  See: https://material-ui.com/guides/minimizing-bundle-size/#how-to-reduce-the-bundle-size